app/src/main/java/org/oppia/app/utility/KeyboardHelper.kt

```diff
@@ -0,0 +1,22 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+
+/** KeyboardHelper help to handle the visiblity of softinputkeybord. */
+class KeyboardHelper {
+  companion object {
+    fun hideSoftKeyboard(view: View, context: Context) {
```

Comment:
Please add a brief KDoc here & for the next method explaining what the method does, and what view/context to supply.

Reply:
Done